.ado/templates/run-windows-with-certificates.yml

```diff
@@ -21,22 +21,33 @@ parameters:
     type: string
   - name: workingDirectory
     type: string
+  - name: restoreLockedMode
+    type: boolean
+    default: true
 
 steps:
-  - task: Cache@2
-    inputs:
-      key: '"run-windows-with-certificates" | "${{ parameters.workingDirectory }}" | ${{ parameters.workingDirectory }}/windows/NuGet.config | **/packages.config | **/packages.lock.json'
-      restoreKeys: '"run-windows-with-certificates" | "${{ parameters.workingDirectory }}" | ${{ parameters.workingDirectory }}/windows/NuGet.config'
-      path: ${{ parameters.workingDirectory }}/windows/packages/
-    displayName: Copy cached NuGet packages
-
   - ${{ if eq(parameters.buildConfiguration, 'Debug') }}:
-    - script: yarn windows --no-packager --no-launch ${{ parameters.deployOption }} --arch ${{ parameters.buildPlatform }} --logging --buildLogDirectory ${{ parameters.buildLogDirectory }}
+    - script: >
+        yarn windows
+        --arch ${{ parameters.buildPlatform }}
+        --no-launch
+        --logging
+        --buildLogDirectory ${{ parameters.buildLogDirectory }}
+        --msbuildprops RestoreLockedMode=${{ parameters.restoreLockedMode }}
+        ${{ parameters.deployOption }}
       displayName: run-windows (Debug)
       workingDirectory: ${{ parameters.workingDirectory }}
 
   - ${{ if and(eq(parameters.buildConfiguration, 'Release'), eq(parameters.buildEnvironment, 'PullRequest')) }}:
-    - script: yarn windows --no-packager --no-launch ${{ parameters.deployOption }} --arch ${{ parameters.buildPlatform }} --logging --buildLogDirectory ${{ parameters.buildLogDirectory }} --release
+    - script: >
+        yarn windows
```

Comment:
just checking if you are intentionally dropping `--no-packager`

Reply:
Yep. `--no-packager` has been implied by `--no-launch` for a few versions.